README.md

```diff
@@ -1,67 +1,34 @@
 # Transformers for Time Series
```

Comment:
Why are we removing such big chunks of the Readme ?

Reply:
The purpose was to simplify the package documentation so that only essential info is there for SW developers.

Reply:
Yes but this content is important, it show how and why I developed that repo. The README is not the documentation, I think we can get away with having a detailed file here.